app/commands/user/reputation_token/award_for_pull_request_author.rb

```diff
@@ -28,7 +28,7 @@ def call
           external_url: params[:html_url],
           merged_at: params[:merged_at]
         )
-        token.update!(level: reputation_level)
+        token&.update!(level: reputation_level)
```

Comment:
I had to modify this as the `User::ReputationToken::Create` call above returns `nil` for the system/ghost user.